packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -923,4 +881,70 @@ export default class Editor {
     }
 
     //#endregion
+
+    //#region Dark mode APIs
+
+        /**
+     * Set the dark mode state and converts the content.

```

Comment:
converts the content is ambiguous. consider something like "converts the editor content to match the requested darkmode state"